mscore/pianoview.cpp

```diff
@@ -561,16 +566,22 @@ void PianoView::wheelEvent(QWheelEvent* event)
 //   showPopupMenu
 //---------------------------------------------------------
 
-void PianoView::showPopupMenu(const QPoint& pos)
+void PianoView::showPopupMenu(const QPoint& posGlobal)
       {
       QMenu popup(this);
-//      popup.addAction(getAction("cut"));
-//      popup.addAction(getAction("copy"));
-      popup.addAction(getAction("paste"));
-//      popup.addAction(getAction("swap"));
+      {
+            QAction* act = new QAction(tr("Copy notes"));
+            connect(act, SIGNAL(triggered(bool)), this, SLOT(copyNotes()));
+            popup.addAction(act);
+      }
+      {
+            QAction* act = new QAction(tr("Paste notes here"));
+            connect(act, SIGNAL(triggered(bool)), this, SLOT(pasteNotesAtCursor()));
+            popup.addAction(act);
```

Comment:
You don't have to write like this, just put blank lines between sections.

Reply:
But if, then banner style braces ;-)

Reply:
I used this style so that I don't need to come up with new vairable names for each QAction in every item.  A little easier to maintain if the menu gets extended in the future.

Reply:
not needed though, you can re-use `act`